src/unstract/sdk/index.py

```diff
@@ -357,6 +373,45 @@ def generate_file_id(
         hashed_index_key = ToolUtils.hash_str(json.dumps(index_key, sort_keys=True))
         return hashed_index_key
 
+    @deprecated("Deprecated class and method. Use Index and index() instead")
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change message to `Use index() method of Index instance instead.`.